docs-site/content/en/cli-reference/graph.md

```
@@ -48,6 +50,28 @@ $ moai graph query --milestones-no-card
 
 `--edges <path>` points at a different edges.jsonl, and a positional root argument selects a different project root.
 
+Before answering, a query refreshes the mechanical layers (the @MX index and edges.jsonl) when they have gone stale. Only files whose content hash moved are re-parsed, so uncommitted edits are reflected in the answer without a rescan; a refresh whose measured cost exceeds gate.yaml's `update_budget_ms` (default 2000ms) warns and still answers. Every answer prints the tree root and commit (or dirty fingerprint) it was computed from, so there is no mistaking which tree answered.
+
+## moai graph check
+
+```bash
+$ moai graph check
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)